source/DataAccess/Oracle/OracleDDLHelper.ts

```diff
@@ -18,8 +28,8 @@ export default class OracleDDLHelper implements IDDLHelper {
             .map((name) => {
                 // This apparently does not return metadata for some system tables/views like ALL_TABLES but will return
                 // proper DDL otherwise
-                return `SELECT 1 as "priority", TABLE_NAME, DBMS_METADATA.GET_DDL(\'TABLE\', TABLE_NAME, OWNER) as ddl ` +
-                `FROM ALL_TABLES where TABLE_NAME = \'${name}\' `;
+                return `SELECT ${prioritizedObjects.indexOf(name)} as "priority", TABLE_NAME, DBMS_METADATA.GET_DDL(\'TABLE\', TABLE_NAME, OWNER) as ddl ` +
```

Comment:
Repurposing the 'priority' of each record. At a later time indexes can appear well after all tables.